analysis/import/SonarImporter/src/test/kotlin/de/maibornwolff/codecharta/importer/sonar/model/ErrorResponseTest.kt

```diff
@@ -0,0 +1,33 @@
+package de.maibornwolff.codecharta.importer.sonar.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotEquals
+import org.junit.jupiter.api.Test
+
+class ErrorResponseTest {
+    private val erFull = ErrorResponse(arrayOf(ErrorEntity("stuff"), ErrorEntity("stuff2")))
+    private val erFullMod = ErrorResponse(arrayOf(ErrorEntity("stuff"), ErrorEntity("stuff2"), ErrorEntity("more stuff")))
+    private val erFull2 = ErrorResponse(arrayOf(ErrorEntity("stuff"), ErrorEntity("stuff2")))
+
+    @Test
+    fun `should identify equals correctly`() {
+        val er = ErrorResponse(arrayOf())
+        val er2 = ErrorResponse(arrayOf())
+        val erImage = er
+        val er3: ErrorResponse? = null
```

Comment:
Let's try to not use abbreviations for variable names. That will help future developers a lot I think.

Reply:
Fixed